diff --git a/node_modules/react-native-google-places-autocomplete/GooglePlacesAutocomplete.js b/node_modules/react-native-google-places-autocomplete/GooglePlacesAutocomplete.js
index 3b71f9f..1dc42b5 100644
--- a/node_modules/react-native-google-places-autocomplete/GooglePlacesAutocomplete.js
+++ b/node_modules/react-native-google-places-autocomplete/GooglePlacesAutocomplete.js
@@ -518,6 +518,11 @@ export const GooglePlacesAutocomplete = forwardRef((props, ref) => {
           }
         } else {
           // console.warn("google places autocomplete: request could not be completed or has been aborted");
+          if (!props.onFail)
+            console.warn(`google places autocomplete: ${request.status}: ${request.statusText}. ${request.responseText}`);
+          else {
+            props.onFail(`${request.status}: ${request.statusText}. ${request.responseText}`);
+          }
         }
       };
 
@@ -759,6 +764,7 @@ export const GooglePlacesAutocomplete = forwardRef((props, ref) => {
       return (
         <FlatList
           nativeID='result-list-id'
+          listKey='result-list-id'
           scrollEnabled={!props.disableScroll}
           style={[
             props.suppressDefaultStyles ? {} : defaultStyles.listView,
